mw-webapp/src/component/select/option/Option.tsx

```diff
@@ -0,0 +1,16 @@
+
+interface OptionProps{
+  key: string;
+  value: string;
+  text: string;
+}
+export const Option: React.FC<OptionProps> = (props: OptionProps) => {
+  return(
```

Comment:
add docs to public members and properties

Reply:
done